policyengine_us/variables/gov/states/or/tax/income/credits/wfhdc/or_wfhdc_eligible.py

```diff
@@ -0,0 +1,50 @@
+from policyengine_us.model_api import *
+
+
+class or_wfhdc_eligible(Variable):
+    value_type = bool
+    entity = TaxUnit
+    label = "Eligible for the Oregon working family household and dependent care credit"
+    documentation = "Oregon Working Family Household and Dependent Care Credit household eligibility"
+    definition_period = YEAR
+    reference = (
+        "https://www.oregon.gov/dor/forms/FormsPubs/publication-or-17_101-431_2021.pdf#page=121",
+        "https://law.justia.com/codes/oregon/2021/volume-08/chapter-315/section-315-264/",
+    )
+    defined_for = StateCode.OR
+
+    def formula(tax_unit, period, parameters):
+        # Get the parameter tree for the Oregon WFHDC.
+        p = parameters(period).gov.states["or"].tax.income.credits.wfhdc
+
+        # Get the household size.
+        household_size = tax_unit("tax_unit_size", period)
+
+        # Check that the household size is large enough.
+        size_eligible = household_size >= p.min_household_size
+
+        # Get the income threshold based on household size.
+        fpg = tax_unit("tax_unit_fpg", period)
+        income_threshold = fpg * p.fpg_rate
+
+        # Get household income, the larger of federal and Oregon AGI.
+        federal_agi = tax_unit("adjusted_gross_income", period)
+        or_agi = tax_unit("or_income_after_subtractions", period)
```

Comment:
According to chatgpt, the `or_income_after_subtractions` is the same as `or_agi` . But this would cause circular error because `or_wfhdc` is in the refundable credit list.

Reply:
we will consolidate the Oregon AGI files in [this](https://github.com/PolicyEngine/policyengine-us/issues/3567) issue.